docs/tools/index.md

```diff
@@ -34,26 +34,22 @@ development and testing purposes.
 [Dashboard](/docs/tasks/web-ui-dashboard/), the web-based user interface of Kubernetes, allows you to deploy containerized applications
 to a Kubernetes cluster, troubleshoot them, and manage the cluster and its resources itself. 
 
-## Third-Party Tools
```

Comment:
If the removal is needed, then I guess we also want to modify the below lines at the beginning of this doc.
```
 Kubernetes contains several built-in tools to help you work with the Kubernetes system, and also supports third-party tooling.
 
 ## Native Tools
 
 Kubernetes contains the following built-in tools:
```

Reply:
@xiangpengzhao I also felt the same, was looking for some opinion. I've made those changes.
Do you mind If we keep the line,
> Kubernetes contains several built-in tools to help you work with the Kubernetes system

Reply:
It was exactly what I was thinking of to keep this line :)

Reply:
Sure :smile: